cmd/gnoland/main.go

```diff
@@ -42,19 +35,81 @@ var flags struct {
 	rootDir               string
 }
 
-func runMain(args []string) error {
-	fs := flag.NewFlagSet("gnoland", flag.ExitOnError)
-	fs.BoolVar(&flags.skipFailingGenesisTxs, "skip-failing-genesis-txs", false, "don't panic when replaying invalid genesis txs")
-	fs.BoolVar(&flags.skipStart, "skip-start", false, "quit after initialization, don't start the node")
-	fs.StringVar(&flags.genesisBalancesFile, "genesis-balances-file", "./gnoland/genesis/genesis_balances.txt", "initial distribution file")
-	fs.StringVar(&flags.genesisTxsFile, "genesis-txs-file", "./gnoland/genesis/genesis_txs.txt", "initial txs to replay")
-	fs.StringVar(&flags.chainID, "chainid", "dev", "chainid")
-	fs.StringVar(&flags.rootDir, "root-dir", "testdir", "directory for config and data")
-	fs.StringVar(&flags.genesisRemote, "genesis-remote", "localhost:26657", "replacement for '%%REMOTE%%' in genesis")
-	fs.Parse(args)
+func main() {
+	cfg := &gnolandCfg{}
+
+	cmd := commands.NewCommand(
+		"gnoland",
+		commands.Metadata{
+			Name:       "",
+			ShortUsage: "[flags] [<arg>...]",
+			LongHelp:   "Starts the gnoland blockchain node",
+		},
+		cfg,
+	)
+
+	if err := cmd.ParseAndRun(context.Background(), os.Args[1:]); err != nil {
+		_, _ = fmt.Fprintln(os.Stderr, err.Error())
```

Comment:
```suggestion
		_, _ = fmt.Fprintf(os.Stderr, "%+v", err)
```

Reply:
Changed it:

[4a5ab3d](https://github.com/gnolang/gno/pull/497/commits/4a5ab3df28a002b696c741cd0ad7fe237b2e1d43)